src/stravalib/protocol.py

```diff
@@ -8,11 +8,14 @@
 import abc
 import functools
 import logging
+import os
+import time
 from collections.abc import Callable
 from typing import TYPE_CHECKING, Any, Literal, TypedDict
 from urllib.parse import urlencode, urljoin, urlunsplit
 
 import requests
+from dotenv import load_dotenv
```

Comment:
Note: this is a new dependency, but we could make it optional. Users can always bypass dotenv and just set envt variables themselves too.